src/pages/library/Datasets.js

```diff
@@ -169,7 +170,9 @@ const amppd = () => h(Participant, {
         div({ style: { width: 410 } }, [`• STEADY-PD3 Study`]),
         div(['• SURE-PD3 Study'])
       ])
-    ])
+    ]),
+    p([`Global Parkinson's Genetics Program (GP2) data is accessible through an approved single AMP PD-GP2 DUA, `,
+      h(Link, { href: 'https://www.amp-pd.org/form/update-amp-pd-application', ...Utils.newTabLinkProps }, 'here.')])
```

Comment:
Just to match indentations:
```suggestion
    p([
      `Global Parkinson's Genetics Program (GP2) data is accessible through an approved single AMP PD-GP2 DUA, `,
      h(Link, { href: 'https://www.amp-pd.org/form/update-amp-pd-application', ...Utils.newTabLinkProps }, ['here.'])
    ])
```